### DISCUSSION TITLE
Thank you for publishing to winget-pkgs!

### DISCUSSION THREAD

**Discussion by pem884** [mtkennerly/ludusavi#349] [General]
I saw there's a new version available, and even though it's not on winget yet, I wanted to thank you for publishing the manifest there. Hope this is a good place for that!

I'm wondering offhandedly as I type this if there's a way to automate. I might look at that if I can kick my brain into gear; it'd probably be a github action or something like that. I'm not experienced though.

Anyway - cheers!

**Comment by mtkennerly** [comment]
You're absolutely welcome, and thanks for the nice comment :)

I've briefly looked at [winget-releaser](https://github.com/vedantmgoyal9/winget-releaser), but I haven't tried setting it up yet. We'd need to make sure that it can distinguish the 64-bit and 32-bit builds and that it can update the `ReleaseNotes` correctly.

**Reply by mtkennerly** [reply]
I've *mostly* automated this now via a Python script. There are still a couple of manual bits (inserting the changelog and opening the pull request), but I don't really mind that:

https://github.com/mtkennerly/ludusavi/blob/ae6cc0f74fc475084aec0e0153fc976d036b415d/tasks.py#L182-L196
